gulpfile.js

```diff
@@ -0,0 +1,51 @@
+var path = require('path');
```

Comment:
I am not familiar with gulp, if will be better if you can add some comment to show use cases of these task.